app/src/main/java/co/condorlabs/customcomponents/graphics/barsgraph/BarsGraph.kt

```diff
@@ -0,0 +1,266 @@
+package co.condorlabs.customcomponents.graphics.barsgraph
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.graphics.*
+import android.util.AttributeSet
+import android.view.View
+import co.condorlabs.customcomponents.DEFAULT_STYLE_ATTR
+import co.condorlabs.customcomponents.DEFAULT_STYLE_RES
+import co.condorlabs.customcomponents.R
+import co.condorlabs.customcomponents.graphics.barsgraph.models.barsgraph.Bar
+import co.condorlabs.customcomponents.graphics.barsgraph.models.barsgraph.BarsGraphConfig
+
+/**
+ * Created by E Duque on 3/30/20.
+ * Condor Labs
+ * eduque@condorlabs.io
+ */
+class BarsGraph @JvmOverloads constructor(
+    context: Context?,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0,
+    defStyleRes: Int = 0
+) : View(context, attrs, defStyleAttr, defStyleRes) {
+
+    private var barsGraphConfig: BarsGraphConfig? = null
+    private var numbersTextSize = 0F
+    private var labelsTextSize = 0F
+    private var horizontalLinesStrokeWidth = 0F
+    private var barsStrokeWidth = 0F
+    private var barsMargin = 0F
+    private var barsStrokeWidthMiddle = 0F
+    private var horizontalLinesPaint = Paint().apply {
+        strokeWidth = horizontalLinesStrokeWidth
+        style = Paint.Style.STROKE
+        strokeCap = Paint.Cap.ROUND
+        strokeJoin = Paint.Join.ROUND
+        isAntiAlias = true
+        isDither = true
+    }
+    private var barsPaint = Paint().apply {
+        style = Paint.Style.FILL
+        isAntiAlias = true
+        isDither = true
+    }
+    private var labelsPaint = Paint().apply {
+        style = Paint.Style.FILL
+        strokeCap = Paint.Cap.ROUND
+        strokeJoin = Paint.Join.ROUND
+        isAntiAlias = true
+        isDither = true
+        textSize = labelsTextSize
+        textAlign = Paint.Align.CENTER
+    }
+    private var countLabelsPaint = Paint().apply {
+        style = Paint.Style.FILL
+        strokeCap = Paint.Cap.ROUND
+        strokeJoin = Paint.Join.ROUND
+        isAntiAlias = true
+        isDither = true
+        textAlign = Paint.Align.CENTER
+    }
+    private var numberOfHorizontalLines = 1
+    private var numberOfBars = 0
+    private var rangeValue = 0
+    private val widthPercentageToDrawLinesAndBars = 80F
+    private val barsBorderStrokeWith = 8F
+    private val barsBorderCornerRadius = 20F
+    private val internalBarsBorderCornerRadius = barsBorderCornerRadius - barsBorderStrokeWith
+    private val corners = floatArrayOf(
```

Comment:
Those variables should be at the top because they are global

Reply:
They are at the top, right?